tendrl/monitoring_integration/flows/delete_monitoring_data/__init__.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import json
+import os
+
+from tendrl.commons import flows
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils.time_utils import now as tendrl_now
+from tendrl.monitoring_integration.flows import alert_utils
```

Comment:
use https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/alert_utils.py

Reply:
ok . will try that

Reply:
After re-factoring this is clean now